src/components/ProjectSettingsTabs/PatchAliasesTab/getFormSchema.ts

```diff
@@ -0,0 +1,68 @@
+import { Field } from "@rjsf/core";
+import { SpruceFormProps } from "components/SpruceForm";
+import widgets from "components/SpruceForm/Widgets";
+import { alias, form } from "../utils";
+import { FormState } from "./types";
+
+const { aliasArraySchema, aliasRowUiSchema } = alias;
+const { overrideRadioBox } = form;
+
+export const getFormSchema = (
+  repoData?: FormState
+): {
+  fields: Record<string, Field>;
+  schema: SpruceFormProps["schema"];
+  uiSchema: SpruceFormProps["uiSchema"];
+} => ({
+  fields: {},
+  schema: {
+    definitions: {
+      aliasArray: aliasArraySchema,
+    },
+    type: "object" as "object",
+    properties: {
+      patchAliases: {
+        title: "Patch Aliases",
+        description:
+          "Specify aliases to use with the CLI. Aliases may be specified multiple times. The result will be their union. All regular expressions must be valid Golang regular expressions. Use an alias with the --alias flag passed to the CLI patch command.",
+        ...overrideRadioBox(
+          "aliases",
+          ["Override Repo Patch Aliases", "Default to Repo Patch Aliases"],
```

Comment:
Is there an example of a project that default to a repo where the repo has patch aliases populated?

Reply:
No, but you could add one to the repo that I linked to in the description and then check out the evergreen project's settings (since evergreen defaults to that repo).

Reply:
I noticed that if I had a project that had overridden patch aliases, but then I clicked on "Default to Repo Patch Aliases" and then saved it, that the override wasn't deleted. (idk how to make those videos you do but note that this Save toast is from when I clicked "Default to Repo Patch Aliases" and then saved the page; i.e. the overrides are still there, even after refereshing)
![image](https://user-images.githubusercontent.com/26798134/153956079-723a60c1-fb70-4507-a3b7-a6d2dad3f032.png)

Reply:
Thank you good catch! I've added some tests around this too.